internal/apitest/runner.go

```diff
@@ -198,7 +198,7 @@ func findTests(rootDir string, testMatch []string) ([]string, error) {
 	return tests, nil
 }
 
-func loadTest(testDir string, suiteName string, testName string, tags []string, env map[string]string) (TestRequest, error) {
+func uploadTest(testDir string, suiteName string, testName string, tags []string, env map[string]string) (TestRequest, error) {
```

Comment:
Hm... that'd be a misnomer, since this method isn't uploading anything. It reads local files and creates a request payload.